plugins/node.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import globals from 'globals';
+import nodePlugin from 'eslint-plugin-n';
+
+export default {
+  plugins: {
+    n: nodePlugin
+  },
+  languageOptions: {
+    globals: globals.node
+  },
+  rules: {
+    // enforce `return` after a callback
+    'n/callback-return': 0,
+    // enforce either `module.exports` or `exports`
+    'n/exports-style': 0,
+    // enforce the style of file extensions in `import` declarations
+    'n/file-extension-in-import': 0,
+    // enforce `require()` on top-level module scope
+    'n/global-require': 1,
+    // require error handling in callbacks
+    'n/handle-callback-err': 0,
+    // suggest correct usage of hashbang
+    'n/hashbang': 0,
+    // ensure Node.js-style error-first callback pattern is followed
+    'n/no-callback-literal': 1,
+    // disallow deprecated APIs
+    'n/no-deprecated-api': 2,
+    // disallow the assignment to `exports`
+    'n/no-exports-assign': 2,
+    // disallow `import` declarations which import extraneous modules
+    'n/no-extraneous-import': 0,
+    // disallow `require()` expressions which import extraneous modules
+    'n/no-extraneous-require': 0,
+    // disallow `import` declarations which import non-existence modules
+    'n/no-missing-import': 0,
```

Comment:
Shouldn't that match with `n/no-missing-require': 2`?

```suggestion
    'n/no-missing-import': 2,
```

Reply:
I was thinking the same thing. Your comment is enough to convince me we should change it here. 🙌